### PR TITLE
feat(tocco-ui): map number components

### PR DESCRIPTION
**Pull request by khanlifts**
- update NumberEdit to accept min/max pre/post point
- adjust defaultMapping and typeEditorFactory
- add stories
- add typeEditables

### CODE REVIEW

File: packages/app-extensions/src/formField/typeEditable/decimal.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import _get from 'lodash/get'
+
+export default {
+  getOptions: ({modelField}) => ({
+    postPointDigits: _get(modelField, 'validation.decimalDigits.postPointDigits', null),
+    prePointDigits: _get(modelField, 'validation.decimalDigits.prePointDigits', null),
+    minValue: _get(modelField, 'validation.numberRange.fromIncluding', null),
+    maxValue: _get(modelField, 'validation.numberRange.toIncluding', null),
+    decimalScale: 28,
```
Comment (MethodenMann) [inline packages/app-extensions/src/formField/typeEditable/decimal.js]: where is this number coming from?

File: packages/app-extensions/src/formField/typeEditable/long.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import _get from 'lodash/get'
+
+export default {
```
Comment (MethodenMann) [inline packages/app-extensions/src/formField/typeEditable/long.js]: exactly the same as integer. map long to integer and delete this file

File: packages/app-extensions/src/formField/typeEditable/number.js [hunk 3]
```diff
@@ -5,6 +5,8 @@ export default {
     prePointDigits: _get(modelField, 'validation.decimalDigits.prePointDigits', null),
     postPointDigits: _get(modelField, 'validation.decimalDigits.postPointDigits', null),
     minValue: _get(modelField, 'validation.numberRange.fromIncluding', null),
-    maxValue: _get(modelField, 'validation.numberRange.toIncluding', null)
+    maxValue: _get(modelField, 'validation.numberRange.toIncluding', null),
+    decimalScale: 100,
```
Comment (MethodenMann) [inline packages/app-extensions/src/formField/typeEditable/number.js]: why 100?

File: packages/tocco-ui/src/EditableValue/EditableValue.stories.js [hunk 4]
```diff
@@ -134,11 +134,16 @@ storiesOf('Tocco-UI | EditableValue', module)
     () =>
       <EditableValueStory
         type="integer"
-        options={object('options', {minValue: 0, maxValue: 300})}
+        options={object('options', {
```
Comment (MethodenMann) [inline packages/tocco-ui/src/EditableValue/EditableValue.stories.js]: Delete Integer story. we dont have an integer EditableValue

File: packages/tocco-ui/src/EditableValue/typeEditors/NumberEdit.js [hunk 5]
```diff
@@ -6,53 +6,48 @@ import {parseLocalePlaceholder, convertStringToNumber} from '../utils'
...
+  if (minValue && floatValue < minValue) {
+    return false
+  }
+
+  return true
 }
 
-export const returnAllowedIntegerObject = (minValue, maxValue) => {
-  return {minValue, maxValue}
+export const calculateDecimalScale = (postPointDigits, decimalScale) => {
```
Comment (MethodenMann) [inline packages/tocco-ui/src/EditableValue/typeEditors/NumberEdit.js]: The more i think about it:
decimal scale and postpoint digits are the same, right? If yes, remove decimal scale from component and convert any "decimalScale" to "postPointDigits" in typeEditable file

File: packages/app-extensions/src/formField/typeEditable/decimal.js (hunk 1 above)
Comment (khanlifts) [reply packages/app-extensions/src/formField/typeEditable/decimal.js]: +/-7.92282E+28 is max/min of type decimal.

File: packages/app-extensions/src/formField/typeEditable/number.js (hunk 3 above)
Comment (khanlifts) [reply packages/app-extensions/src/formField/typeEditable/number.js]: With changes from 0a32012 decimalScale can be omitted here.

File: packages/tocco-ui/src/EditableValue/typeEditors/NumberEdit.js (hunk 5 above)
Comment (khanlifts) [reply packages/tocco-ui/src/EditableValue/typeEditors/NumberEdit.js]: I don't think that is an improvement. As minValue/maxValue is more important there are inputs without postPointDigits. In these cases we control the number of decimals with decimalScale.

File: packages/tocco-ui/src/EditableValue/EditableValue.stories.js [hunk 6]
```diff
@@ -129,16 +129,8 @@ storiesOf('Tocco-UI | EditableValue', module)
...
-    'Integer',
-    () =>
-      <EditableValueStory
-        type="integer"
-        options={object('options', {minValue: 0, maxValue: 300})}
-        defaultValue={200}
-        knobType={number}
-      />
-  ).add(
+  )
```
Comment (MethodenMann) [inline packages/tocco-ui/src/EditableValue/EditableValue.stories.js]: purposely?

Comment (khanlifts) [reply packages/tocco-ui/src/EditableValue/EditableValue.stories.js]: no

**Pull request merged**